src/go/rpk/pkg/cli/container/start.go

```diff
@@ -584,10 +584,7 @@ func checkConsole(node node) func() error {
 			return fmt.Errorf("error checking console status: %v", err)
 		}
 		if res["isHttpOk"] {
```

Comment:
Similar to k8s, we can just check for the returned http status code. The response body is more informational for those who are curious why it's not reporting 200.